File: test/fixtures/vulnerable/crypto_usage.js
const crypto = require('crypto');
const forge = require('node-forge');

// RSA key generation - QUANTUM VULNERABLE
const { publicKey, privateKey } = crypto.generateKeyPairSync('rsa', {
  modulusLength: 2048,
});

// DES encryption - WEAK
const cipher = crypto.createCipheriv('des', key, iv);

// MD5 hashing - WEAK
const hash = crypto.createHash('md5');

File: test/fixtures/vulnerable/crypto_usage.rs
use rsa::{RsaPrivateKey, RsaPublicKey};
use rand::rngs::OsRng;

fn main() {
    // RSA - QUANTUM VULNERABLE
    let private_key = RsaPrivateKey::new(&mut OsRng, 2048).unwrap();
    let public_key = RsaPublicKey::from(&private_key);
    println!("{:?}", public_key);
}
